docs/getting_started/CONTRIBUTING.rst

```diff
@@ -66,6 +66,20 @@ the semantic versioning website.
 
 For more info on versions see http://semver.org
 
+Releasing to PyPI
+-----------------
+
+A new release of SMRF will be pushed to the `smrf-dev <https://pypi.org/project/smrf-dev/>` development package on Pypi. This will
+not create a stable release and is intended on use by developers.
+
+To create a new release on `Pypi.org <https://pypi.org/>`_, follow these steps:
+
+#. Create a new release for `smrf`
+#. Name the tag and release the version number, for example `v0.12.0 <https://github.com/USDA-ARS-NWRC/smrf/releases/tag/v0.11.0>`_
```

Comment:
Is this intentionally pointing to the example in WFR?

Reply:
Where is it pointing to that? I see the link as SMRF.